zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```
@@ -733,7 +733,58 @@ paths:
             application/problem+json:
               schema:
                 $ref: "#/components/schemas/ProblemDetail"
-
+  /variables/{variableKey}:
+    get:
+      tags:
+        - Variable
+      summary: Return variable by a variable key.
```

```suggestion
      summary: Return variable by a variable key
```

:x: `summary` property of an endpoint turns into the left navigation in the docs; they should not end in a period.